From 8fc93b8c0349447e22c93c3c9eca1c8441fd32a5 Mon Sep 17 00:00:00 2001
From: MouriNaruto <Mouri_Naruto@Outlook.com>
Date: Sun, 11 Dec 2022 01:22:23 +0800
Subject: [PATCH] Adjust VC-LTL target platform version.

---
 cmake/VC-LTL.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/cmake/VC-LTL.cmake b/cmake/VC-LTL.cmake
index 2bab2f21d6..e0a08af684 100644
--- a/cmake/VC-LTL.cmake
+++ b/cmake/VC-LTL.cmake
@@ -28,7 +28,7 @@ set(__VC_LTL_CMAKE_INCLUDE_GUARD 1)
 
 #控制TargetPlatform版本，目前可用版本为5.1.2600.0     6.0.6000.0（默认）    6.2.9200.0     10.0.10240.0    10.0.19041.0
 if(NOT DEFINED WindowsTargetPlatformMinVersion)
-    set(WindowsTargetPlatformMinVersion "10.0.19041.0" CACHE STRING "" FORCE)
+    set(WindowsTargetPlatformMinVersion "6.0.6000.0" CACHE STRING "" FORCE)
 endif()
 
 #启用干净的导入表，消除 ucrt apiset(如：api-ms-win-crt-time-l1-1-0.dll)，满足强迫症患者。
-- 
2.38.1.windows.1

